docs/src/pages/faq.tsx

```diff
@@ -50,6 +50,26 @@ export default function FAQ() {
           scenarios, producing works that significantly contribute to the
           community and pave the way for future advancements.
         </p>
+        <h3>How to fix an issue on OpenDevin?</h3>
+        <p>
+          To fix an issue on GitHub using OpenDevin, send a prompt to OpenDevin asking it to follow these steps:
+          <ol>
+            <li>Read the issue on <a href="https://github.com/OpenDevin/OpenDevin/issues/1611">GitHub</a></li>
+            <li>Clone the repository and check out a new branch</li>
+            <li>Based on the instructions in the issue description, modify files to fix the issue</li>
+            <li>Push the resulting output to GitHub using the GITHUB_TOKEN environment variable</li>
+            <li>Tell me the link that I need to go to to send a pull request</li>
+          </ol>
+          Before you run OpenDevin, you can do:
+          <pre>
+            export SANDBOX_ENV_GITHUB_TOKEN=XXX
```

Comment:
Is this actually implemented yet?

Reply:
Yes, it is.

Reply:
`github_token` is the only one i can find -- not sure if `SANDBOX_ENV_GITHUB_TOKEN` will be actually set to `github_token` -- if so, i think i'm good with this one :D

https://github.com/OpenDevin/OpenDevin/blob/50c2141d2094c287f2bbd16d64ff482b96cd0e60/opendevin/core/config.py#L95

Reply:
Bc i didn't actually see a prefix with `sandbox_env` in the `config.py`

Reply:
I believe env vars that start with SANDBOX_ENV_ are read from the env, bypassing the usual need to be in config.py
Update: found it https://github.com/OpenDevin/OpenDevin/blob/50c2141d2094c287f2bbd16d64ff482b96cd0e60/opendevin/runtime/sandbox.py#L16

Reply:
@enyst ahhh got it! Thanks a ton!